rs/boundary_node/rate_limits/canister_client/ratelimits.yml

```diff
@@ -0,0 +1,13 @@
+- canister_id: ryjl3-tyaaa-aaaaa-aaaba-cai
```

Comment:
Just a remark regarding a potential problem of setting rate-limits on the `rate-limit` canister itself. We shouldn't accidentally block requests to this canister, as this will be an unrecoverable operation, i guess.